packages/amplify-graphql-api-construct/src/graphql-output.ts

```diff
@@ -14,6 +14,7 @@ export const graphqlOutputSchemaV1 = z.object({
     awsAppsyncAuthenticationType: AwsAppsyncAuthenticationType,
     awsAppsyncApiKey: z.string().optional(),
     awsAppsyncApiId: z.string(),
+    amplifyApiModelSchemaS3Uri: z.string(),
```

Comment:
We might be able to use zod transforms to handle the fetching. https://github.com/colinhacks/zod#transform

cc @edwardfoyle

Reply:
I'd rather leave the data types here straightforward and clear, and rely on the tooling to perform the necessary steps to fully reconstruct the codegen state.